modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -132,33 +132,86 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
         'pass' => $verification,
       );
 
-      if ($res) {
+      // Check if the user is blocked.
+      $block_codes = array(
+        'z305-delinq-1' => 'z305-delinq-n-1',
+        'z305-delinq-2' => 'z305-delinq-n-2',
+        'z305-delinq-3' => 'z305-delinq-n-3',
+      );
+
+      // Loop through denlinq-1, denlinq-2, denlinq-3.
```

Comment:
Just "sub libraries" otherwise you have to update the comment each time you alter the list